database/javascript/2017/16065.yaml

```diff
@@ -9,8 +9,6 @@ references:
 affected:
     - name: openssl.js
       version:
-        - "==1.0.2"
-        - "==1.0.1"
+        - "<=0.0.2-security"
       fixedin:
-        - "==0.0.1-security"
```

Comment:
this looks correct. Why removed it. Same for another version

Reply:
But according to NPM registry if we give latest version it will get all prior versions.

Reply:
This looks ok to me based on our discussion.